setup.py

```diff
@@ -524,15 +532,15 @@ def get_sphinx_theme_version() -> str:
 
 # Dict of all providers which are part of the Apache Airflow repository together with their requirements
 PROVIDERS_REQUIREMENTS: Dict[str, List[str]] = {
-    'airbyte': [],
+    'airbyte': http,
```

Comment:
Same for airbyte, apache.livy, and opsgenie as for slack -- we should depend upon the http provider, not http deps directly.

I can't remember where we define cross-provider deps though.

Reply:
Ah, dependencies.json

Reply:
```suggestion
    'airbyte': [],
```

Already depends on http provider

Reply:
Not really. Those are optional dependencies (in extras). Cross-provider dependencies are extra/optional ones (because ofen those are optional dependencies of only one operator or so (like in case of slack). Before that, when http was mandatory, it did not matter. But now we have to explicitly change it to 'apache-airflow-provider-http` actuatlly if we want to be 100% correct.